<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArrayMenu</class>
 <widget class="QWidget" name="ArrayMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="backBtn">
     <property name="text">
      <string>Вернуться в меню</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Диапазон значений</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QSlider" name="maxSlider">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLCDNumber" name="maxLcd"/>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="minLcd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>max</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>min</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSlider" name="minSlider">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Сброс</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Параметры расчёта</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="maxRB">
        <property name="text">
         <string>Наибольшее значение</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="minRB">
        <property name="text">
         <string>Наименьшее значение</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="averageRB">
        <property name="text">
         <string>Среднее арифметическое</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QTextBrowser" name="resultBrowser"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="setButton">
     <property name="text">
      <string>Задать</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTextBrowser" name="startBrowser"/>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QPushButton" name="resultButton">
     <property name="text">
      <string>Вычислить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
